prusti-tests/tests_old/test.rs

```diff
@@ -10,16 +10,16 @@ use std::{
 
 fn get_prusti_rustc_path() -> PathBuf {
     let local_prusti_rustc_path: PathBuf = if cfg!(windows) {
-        ["target", "debug", "prusti-rustc.exe"].iter().collect()
+        ["target", "debug", "prusti-rustc.exe)].iter().collect()
```

Comment:
All of the replacements in this file are wrong.

Reply:
Uh, what a mess. Thanks! I'll just delete this file, as it's never used.